lib/wibox/widget/slider.lua

```diff
@@ -307,7 +323,47 @@ for prop in pairs(properties) do
     end
 end
 
+--- Set the handle widget
```

Comment:
Those `@method` sections are not required. The `@property` is enough. Otherwise, it will document how to set the widget twice. We only document some legacy setters which had more than 1 parameters. Everything else is documented using the property mechanism.